src/OrchardCore.Modules/OrchardCore.Users/Views/EmailAuthenticatorValidation.cshtml

```diff
@@ -0,0 +1,31 @@
+<h5 class="text-center mb-3">@T["Please <a href=\"#\" id=\"RequestCode\">click here</a> to receive a one-time verification code via email."]</h5>
+
+<div class="text-center alert d-none" id="RequestCodeFeedback">
+</div>
+
+<script at="Head" asp-name="RequestVerificationCodeViaEmail">
+    document.addEventListener('DOMContentLoaded', () => {
+
+        const box = document.getElementById('RequestCodeFeedback');
+        const btn = document.getElementById('RequestCode');
+
+        btn.addEventListener('click', (e) => {
+            e.preventDefault();
+            box.classList.add('d-none')
+
+            fetch('@Url.ActionLink("GetCode", "EmailAuthenticator")')
```

Comment:
inject in hidden tag's attribute to prevent javascript injections (or broken links)